### PR TITLE
Add more info about Qt in BUILDING.md

### PR DESCRIPTION
**Pull request by Hasster1**
Specify that Qt needs a "qtmultimedia" module and provide a link to an alternative download

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following tools are required to build RPCS3 on Windows 10 or later:
     with standalone **CMake** tool.
 
 - [Python 3.6+](https://www.python.org/downloads/) (add to PATH)
-- [Qt 6.7.3](https://www.qt.io/download-qt-installer)
+- [Qt 6.7.3](https://www.qt.io/download-qt-installer) with the "qtmultimedia" module (In case you can't download from the official site, you can use [Another Qt installer](https://github.com/miurahr/aqtinstall))
```
Comment (elad335) [inline BUILDING.md:22]: Its in the components of the installer

Comment (Hasster1) [reply BUILDING.md:22]: > Its in the components of the installer

I guess i should specify that it's for those, who don't use the official installer - when i installed qt through aqt, i had to manually add the qtmultimeida module.

Comment (elad335) [reply BUILDING.md:22]: It is in the official I am pretty sure, there is a dropdown options for components types, I installed 6.7.3 today

Comment (elad335) [reply BUILDING.md]: There is no harm in linking an alternative installer I guess

**Review by elad335 — APPROVED** [review]

**Pull request merged**